text/info.py

```diff
@@ -88,3 +89,24 @@ def _plot(counter: Counter, fig_path: str):
         ax.plot(x, y, "s")
 
         fig.savefig(fig_path, bbox_inches="tight")
+
+
+class CountLinesJob(Job):
+    def __init__(self, input_text: tk.Path):
+        self.input_text = input_text
+
+        self.out_num_sentences = self.output_var("num_sentences.txt")
+
+        self.rqmt = {"cpu": 1, "mem": 1, "time": 1}
+
+    def tasks(self):
+        yield Task("run", rqmt=self.rqmt)
+
+    def run(self):
+        zcat_cmd = ["zcat", "-f", "ted2_transcripts.gz"]
```

Comment:
I think this should rather be the input text, otherwise the job would be oddly specific ;)

Reply:
hahaha ups that is from my testing xD